cryptofeed/deribit/deribit.py

```diff
@@ -0,0 +1,205 @@
+                                     bid=msg["params"]["data"]['best_bid_price'],
+                                     ask=msg["params"]["data"]['best_ask_price'])
+
+    async def subscribe(self, websocket):
+        self.websocket = websocket
+        self.__reset()
+        client_id = 0
+
+        for chan in self.channels if self.channels else self.config:
+            for pair in self.pairs if self.pairs else self.config[chan]:
+                client_id += 1
+                await websocket.send(json.dumps(
+                    {
+                        "jsonrpc": "2.0",
+                        "id": client_id,
+                        "method": "public/subscribe",
+                        "params": {
+                            "channels": [
+                                f"{chan}.{pair}.raw"
+                            ]
+                        }
+                    }
+                ))
+
+    async def _book_snapshot(self, msg):
+        timestamp = msg["params"]["data"]["timestamp"]
+        self.l2_book[msg["params"]["data"]["instrument_name"]] = {
+            BID: sd({
+                Decimal(price): Decimal(amount)
+                # _ is always 'new' for snapshot
+                for _, price, amount in msg["params"]["data"]["bids"]
+            }),
+            ASK: sd({
+                Decimal(price): Decimal(amount)
+                for _, price, amount in msg["params"]["data"]["asks"]
+            })
+        }
+
+        await self.book_callback(msg["params"]["data"]["instrument_name"], L2_BOOK, True, None, timestamp)
+
+    async def _book_update(self, msg):
+        timestamp = msg["params"]["data"]["timestamp"]
+        delta = {BID: [], ASK: []}
+
+        for action, price, amount in msg["params"]["data"]["bids"]:
+            bidask = self.l2_book[msg["params"]
+                                  ["data"]["instrument_name"]][BID]
+            if action != "delete":
+                bidask[price] = Decimal(amount)
+                delta[BID].append((Decimal(price), Decimal(amount)))
+            else:
+                del bidask[price]
+                delta[BID].append((Decimal(price), Decimal(amount)))
+
+        for action, price, amount in msg["params"]["data"]["asks"]:
+            bidask = self.l2_book[msg["params"]
+                                  ["data"]["instrument_name"]][ASK]
+            if action != "delete":
+                bidask[price] = amount
+                delta[ASK].append((Decimal(price), Decimal(amount)))
+            else:
+                del bidask[price]
+                delta[ASK].append((Decimal(price), Decimal(amount)))
+        await self.book_callback(msg["params"]["data"]["instrument_name"], L2_BOOK, False, delta, timestamp)
+
+    async def message_handler(self, msg):
+        # Converting msg we got from str to dict using yaml, because it contains singular quotes ' '
+        msg_dict = yaml.safe_load(msg)
```

Comment:
dont use yaml, just use json.loads here. json parsing is significantly faster than yaml, even if you were using the C loader (which we are not). I tested it, and it works just fine with `json.loads`